server/src/services/animal-interest-point.service.ts

```diff
@@ -7,27 +7,23 @@ export class AnimalInterestPointService {
 
     constructor(private readonly databaseService: DatabaseService) { }
 
-    addAnimalIntretPoint(): boolean {
+    addAnimalInterestPoint(): boolean {
 
         const con = this.databaseService.getConnection();
-        let addAnimal = con.then(async (data) => {
-            let animalIntrestPoints = new AnimalInterestPoint();
-
-            animalIntrestPoints.name = "saltLick";
-            animalIntrestPoints.Point_description = "Saltlick at dam point A";
-            animalIntrestPoints.longatude = "28.282984";
-            animalIntrestPoints.latitude = "-25.865828";
-            return data.manager.save(animalIntrestPoints).then(animalIntrestPoints => { console.log("Saved a new animal intrestpoint with id: " + animalIntrestPoints.id) });
+        const addAnimal = con.then(async (data) => {
```

Comment:
This seems messy.

Maybe something along the lines of 
```
const con = await this.databaseService.getConnection();
const animalInterestPoint = new AnimalInterestPoint();
...
const addedAnimal = await con.manager.save(animalInterestPoint);
```
would be better (get rid of .then and make singular instead of plural)?

Reply:
addAnimalInterestPoint(): boolean {
        const con = await this.databaseService.getConnection();
        const animalInterestPoints = new AnimalInterestPoint();

        animalInterestPoints.name = 'saltLick';
        animalInterestPoints.pointDescription = 'Saltlick at dam point A';
        animalInterestPoints.longitude = '28.282984';
        animalInterestPoints.latitude = '-25.865828';
            // tslint:disable-next-line:no-console
        const addedAnimal = await con.manager.save(animalInterestPoints);
        console.log('Saved a new animal interest point with id: ' + animalInterestPoints.id);
        return addedAnimal != null;
    }


like this? 
also it says I shoulf make this function async?